hazelcast-spring/src/test/java/com/hazelcast/spring/TestClientApplicationContext.java

```diff
@@ -102,9 +103,12 @@
     @Resource(name = "client8")
     private HazelcastClientProxy client8;
 
-    @Resource(name = "user-code-deployment-test")
+    @Resource(name = "client9-user-code-deployment-test")
```

Comment:
I tried the break the wheel there by avoiding numeric names, but using self explanatory names. 
It seems you are not letting me to :)